api/sheets/get_property.md

```diff
@@ -1,70 +1,90 @@
+![Retrieve the Common Name of a Country by its DCID](/assets/images/sheets/sheets_get_property_ivory_coast.png)
 
-## Example requests and responses
+### Example 2: Retrieve the Order to which the Plant Austrobaileya Scandens Belongs
 
-Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
+To retrieve the order to which the plant *Austrobaileya Scandens* belongs, perform the following steps:
 
-### Example 1: Retrieve the common names of a country by its `DCID`.
+1. Place your cursor in the cell where you want to add a DCID. In this case, cell A1.
+2. Enter "dc/bsmvthtq89217" for the plant *Austrobaileya Scandens*.
+3. Place your cursor in cell B2 and enter `=DCPROPERTY(A1, "order")`.
+4. Press **Enter** and the order *Austrobaileyales* appears in cell B2.
 
-![](/assets/images/sheets/sheets_get_property_ivory_coast.png)
+![Retrieve the Order to which the Plant Austrobaileya Scandens Belongs](/assets/images/sheets/sheets_get_property_austrobaileyales_order.png)
 
-### Example 2: Retrieve the order to which the plant _Austrobaileya scandens_ belongs.
+### Example 3: Retrieve the Addresses of Stuyvesant High School in New York and Gunn High School in California
 
-![](/assets/images/sheets/sheets_get_property_austrobaileyales_order.png)
+To retrieve the addresses of Stuyvesant High School in New York and Gunn High School in California, perform the following steps:
 
-### Example 3: Retrieve the addresses of Stuyvesant High School in New York and Gunn High School in California.
+1. Place your cursor in cell A1 and enter "nces/360007702877" for *Stuyvesant Hight School in New York*.
+2. Place your cursor in cell A2 and enter "nces/062961004587" for *Gunn High School in California*.
+3. Enter the formula `=DCPROPERTY(A1:A2,"address")` into cell B2 and the addresses of both high schools are populated in column B.
 
-![](/assets/images/sheets/sheets_get_property_school_addresses.png)
+![Retrieve the Addresses of Stuyvesant High School in New York and Gunn High School in California](/assets/images/sheets/sheets_get_property_school_addresses.png)
 
-## Error Returns
 
-If you pass a nonexistent property, an empty value is returned:
+## Error Responses
 
-![](/assets/images/sheets/sheets_get_property_bad_property.png)
+The `=DCPROPERTY(dcids, property)` returns the value of the property label for the specified DCIDs. See the Examples section above for examples of positive responses. If you pass a nonexistent property, an empty value is returned: for example, because the “nonexistent property” does not exist, no value is returned to cell B1 in the following sheet:
```

Comment:
Nit: remove double colon in this sentence.

```suggestion
The `=DCPROPERTY(dcids, property)` returns the value of the property label for the specified DCIDs. See the Examples section above for examples of positive responses. If you pass a nonexistent property, an empty value is returned. For example, because the “nonexistent property” does not exist, no value is returned to cell B1 in the following sheet:
```